cmd/drafter-forwarder/main.go

```diff
@@ -89,37 +89,35 @@ func main() {
 		},
 	)
 
-	if forwardedPorts.Wait != nil {
-		defer func() {
-			defer handlePanics(true)()
+	defer func() {
+		defer goroutineManager.CreateForegroundPanicCollector()()
 
-			if err := forwardedPorts.Wait(); err != nil {
-				panic(err)
-			}
-		}()
-	}
+		if err := forwardedPorts.Wait(); err != nil {
+			panic(err)
+		}
+	}()
 
 	if err != nil {
 		panic(err)
 	}
 
 	defer func() {
-		defer handlePanics(true)()
+		defer goroutineManager.CreateForegroundPanicCollector()()
 
 		if err := forwardedPorts.Close(); err != nil {
 			panic(err)
 		}
 	}()
```

Comment:
That could be an option in the future! I think keeping the panic handler around is a good idea (after all not everywhere we use this returns an error - e.g. Silo's hooks) but as an additional handler it could be useful - maybe even by wrapping the entire function, e.g. with `StartErrorGoroutine(func() error)`, too